force-app/tests/Collection Recipes/IterationRecipes_Tests.cls

```diff
@@ -56,13 +56,14 @@ public class IterationRecipes_Tests {
         try {
             Test.startTest();
             IterationRecipes.iterableApiClientRecipe();
-            System.assert(false, 'Expected IterableApiException');
+            System.Assert.fail('Expected IterableApiException');
```

Comment:
Great use of `Assert.fail` @msrivastav13 
Can you run a search on the entire codebase for hardcoded `Assert.isTrue(true` and `Assert.isFalse(false` and replace them with `Assert.fail` when appropriate?
I saw a couple of cases where this would apply.

Reply:
@pozil It does make sense for some scenarios but there is nothing called Assert.isPass for replacing Assert.isTrue(true). 

Where you thinking of negating it? Like !Assert.Fail? Assert.Fail produces a fatal code and I think we should only use where we are intentional about failing the code.

Reply:
I agree, we should only use this when we want to fail the test (generally in the context of an exception test).